config/feed-variants/20221004-variant-b.json

```diff
@@ -0,0 +1,158 @@
+{
+  "max_days_since_published": 15,
+  "description": "Copy of 20221004-variant-b but with matching_positive_tags_intersection_count present in addition to the points sum",
```

Comment:
Likely a typo that makes no meaningful difference, but this should be `Copy of 20221004-variant-a`, right?

Reply:
Good catch